tests/main/interfaces-content-empty-content-attr/task.yaml

```diff
@@ -0,0 +1,37 @@
+summary: Ensure that the content sharing interface with defaults work.
+
+prepare: |
+    echo "Given a snap declaring a content sharing slot is installed"
+    snap install --edge test-snapd-content-slot-empty-content-attr
+
+    echo "And a snap declaring a content sharing plug is installed"
+    snap install --edge test-snapd-content-plug-empty-content-attr
+
+execute: |
+    CONNECTED_PATTERN="test-snapd-content-slot-empty-content-attr:shared-content +test-snapd-content-plug-empty-content-attr"
+    DISCONNECTED_PATTERN="(?s).*?test-snapd-content-slot-empty-content-attr:shared-content +-.*?- +test-snapd-content-plug-empty-content-attr"
+
+    echo "Then the snap is listed as connected"
+    snap interfaces | grep -Pzq "$CONNECTED_PATTERN"
```

Comment:
Unfortunately the options are not compatible, match does not do it here. Should be something we chase in master too, I agree that MATCH is much nicer here.